### PR TITLE
update rke2-calico to v3.24.5

### PR DESCRIPTION
**Pull request by thomasferrandiz**

### CODE REVIEW

**Review by brandond — APPROVED** [review]

File: packages/rke2-calico/package.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-url: https://github.com/projectcalico/calico/releases/download/v3.24.1/tigera-operator-v3.24.1.tgz
-packageVersion: 03
```
Comment (manuelbuil) [inline packages/rke2-calico/package.yaml:2]: If there is a new url, the packageVersion moves back to 01

File: packages/rke2-calico/package.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+url: https://github.com/projectcalico/calico/releases/download/v3.24.5/tigera-operator-v3.24.5.tgz
+packageVersion: 04
```
Comment (rbrtbnfgl) [inline packages/rke2-calico/package.yaml:2]:
```suggestion
packageVersion: 01
```
When there is a version update the package version counter is restarted.

File: packages/rke2-calico/package.yaml (hunk 1 above)
Comment (thomasferrandiz) [reply packages/rke2-calico/package.yaml:2]: done, thanks

**Review by manuelbuil — APPROVED** [review]

**Comment by brandond** [comment]
@thomasferrandiz do you have permission to merge this now that it's approved?

**Comment by thomasferrandiz** [comment]
yes, I can merge it

**Pull request merged**